contracts/LivepeerProtocol.sol

```diff
@@ -498,6 +508,12 @@ contract LivepeerProtocol {
             isCurrentActiveTranscoder[currentActiveTranscoders[i].id] = true;
             // Set index position of node in current active transcoder set
             currentActiveTranscoderPositions[currentActiveTranscoders[i].id] = i;
+            // Set pending blockRewardCut as actual value
+            transcoders[currentActiveTranscoders[i].id].blockRewardCut = transcoders[currentActiveTranscoders[i].id].pendingBlockRewardCut;
+            // Set pending feeShare as actual value
+            transcoders[currentActiveTranscoders[i].id].feeShare = transcoders[currentActiveTranscoders[i].id].pendingFeeShare;
+            // Set pending pricePerSegment as actual value
+            transcoders[currentActiveTranscoders[i].id].pricePerSegment = transcoders[currentActiveTranscoders[i].id].pendingPricePerSegment;
```

Comment:
There's a lot of reptition in these 3 assignments of indexing into the transcoders array. Should we pull it out into a temporary transcoder variable, and then write it back? Could be more efficient with gas and easier to read.